mix.exs

```diff
@@ -55,7 +55,7 @@ defmodule Appsignal.Phoenix.MixProject do
       end
 
     [
-      {:appsignal_plug, ">= 2.0.8 and < 3.0.0"},
+      {:appsignal_plug, ">= 2.0.9 and < 3.0.0"},
```

Comment:
The lower bound bump is for the transitive dependency `appsignal` version which provides `Appsignal.Utils`.
(btw maybe it should be a direct dependency instead)

Reply:
Thanks! This makes sure the dependency for AppSignal is above 2.0.8 (specifically 2.0.10), which adds `Appsignal.Utils.compile_env`.